panimg/image_builders/dicom.py

```diff
@@ -62,7 +62,7 @@ def _get_headers_by_study(files: Set[Path]):
     grouped by study id.
     """
     studies = {}
-    errors = {}
+    errors = defaultdict(list)
```

Comment:
I've only added those type of annotations where necessary. It's not needed there, mypy is able to infer the type from the assignment a little further down, it raises an error otherwise.